src/patternfly/utilities/Text/text.scss

```diff
@@ -0,0 +1,47 @@
+// stylelint-disable
+
+// Font size options
+$pf-u-font-size-options: (
+  font-size-xs:  (font-size var(--pf-global--FontSize--xs)),
+  font-size-sm:  (font-size var(--pf-global--FontSize--sm)),
+  font-size-md:  (font-size var(--pf-global--FontSize--md)),
+  font-size-lg:  (font-size var(--pf-global--FontSize--lg)),
+  font-size-xl:  (font-size var(--pf-global--FontSize--xl)),
+  font-size-2xl: (font-size var(--pf-global--FontSize--2xl)),
+  font-size-3xl: (font-size var(--pf-global--FontSize--3xl)),
+  font-size-4xl: (font-size var(--pf-global--FontSize--4xl))
+);
+
+@include pf-utility-builder($pf-u-font-size-options, $pf-global--breakpoint-list);
+
+
+// Font weight options
+$pf-u-font-weight-options: (
+  font-weight-light:              (font-weight var(--pf-global--FontWeight--light)),
+  font-weight-normal:             (font-weight var(--pf-global--FontWeight--normal)),
+  font-weight-bold:               (font-weight var(--pf-global--FontWeight--bold)),
+);
+
+@include pf-utility-builder($pf-u-font-weight-options, $pf-global--breakpoint-list);
+
+
+// Font color options
+$pf-u-font-color-options: (
+  color-100: (color var(--pf-global--Color--100)),
+  color-200: (color var(--pf-global--Color--200)),
+  color-300: (color var(--pf-global--Color--300)),
+  color-400: (color var(--pf-global--Color--400)),
+);
+
+@include pf-utility-builder($pf-u-font-color-options, $pf-global--breakpoint-list);
+
+// Font layout options
+$pf-u-font-alignment-options: (
+  font-break-word: (word-break break-word),
+  font-nowrap:     (white-space nowrap),
+  font-wrap:       (white-space normal),
```

Comment:
Were you able to get truncate working?

Reply:
@mcoker just committed with @mattnolting 's suggested `pf-u-text-truncate` utility class:

```
.pf-u-text-truncate {
  min-width: 0 !important;
  max-width: 100% !important;
  overflow: hidden !important;
  text-overflow: ellipsis !important;
  white-space: nowrap !important;
}
```

Looks like the truncation aspect works, but the ellipsis is missing. I thought it was because the width is constrained here by percent rather than pixel, but after changing max-width to `100px` to test, the ellipsis is still MIA. Do you guys have any other ideas why this might be happening?

Reply:
`pf-u-truncate` needs to be applied to the individual element (`div, p, h1-2-3..., etc`) rather than the parent div.

Reply:
@mattnolting totally makes sense - thank you!